src/stores/attachment_store.js

```diff
@@ -29,11 +29,11 @@ class AttachmentsStore extends Store {
     });
   }
 
-  getAttachment(commentId) {
-    var attachment = assign({}, attachments.get(commentId));
+  getAttachments(commentId) {
+    var a = assign({}, attachments.get(commentId));
     attachments = attachments.delete(commentId);
```

Comment:
It is really strange -- I'll find a better way to make sure that we don't get the same attachment twice.